src/DIRAC/Interfaces/API/Job.py

```diff
@@ -82,12 +82,6 @@ def __init__(self, script=None, stdout="std.out", stderr="std.err"):
         self.wfArguments = {}
         self.parametricWFArguments = {}
 
-        # loading the function that will be used to determine the platform (it can be VO specific)
```

Comment:
@fstagni I checked that the getDIRACPlatforms method wasn't overriden by LHCbDIRAC, it's currently not the case.

Reply:
Careful: https://gitlab.cern.ch/lhcb-dirac/LHCbDIRAC/-/blob/master/src/LHCbDIRAC/ConfigurationSystem/Client/Helpers/Resources.py

Reply:
This change is being discussed in #7018.